src/main/java/io/github/nucleuspowered/nucleus/modules/environment/commands/AddTimeCommand.java

```diff
@@ -0,0 +1,50 @@
+package io.github.nucleuspowered.nucleus.modules.environment.commands;
+
+import io.github.nucleuspowered.nucleus.Nucleus;
+import io.github.nucleuspowered.nucleus.Util;
+import io.github.nucleuspowered.nucleus.argumentparsers.WorldTimeArgument;
+import io.github.nucleuspowered.nucleus.internal.annotations.command.Permissions;
+import io.github.nucleuspowered.nucleus.internal.annotations.command.RegisterCommand;
+import io.github.nucleuspowered.nucleus.internal.command.AbstractCommand;
+import io.github.nucleuspowered.nucleus.internal.docgen.annotations.EssentialsEquivalent;
+import org.spongepowered.api.command.CommandResult;
+import org.spongepowered.api.command.CommandSource;
+import org.spongepowered.api.command.args.CommandContext;
+import org.spongepowered.api.command.args.CommandElement;
+import org.spongepowered.api.command.args.GenericArguments;
+import org.spongepowered.api.event.cause.Cause;
+import org.spongepowered.api.text.Text;
+import org.spongepowered.api.util.annotation.NonnullByDefault;
+import org.spongepowered.api.world.storage.WorldProperties;
+import java.util.function.LongFunction;
+
+@Permissions(prefix = "time")
+@RegisterCommand(value = "add", subcommandOf = TimeCommand.class, rootAliasRegister = { "addtime", "timeadd" })
+@EssentialsEquivalent(value = {"time", "day", "night"}, isExact = false, notes = "A time MUST be specified.")
```

Comment:
Remove this annotation - this is for the main command is purely for documentation only! Same for remove command.